internal/cmd/protoc-gen-go-flipt-sdk/main.go

```diff
@@ -234,27 +234,36 @@ type ClientTokenProvider interface {
 // lifecycle support.
 type SDK struct {
 	transport Transport
-    tokenProvider ClientTokenProvider
+    authenticationProvider ClientAuthenticationProvider
```

Comment:
Take it or leave it (go fmt will sort it out either way):
```suggestion
	authenticationProvider ClientAuthenticationProvider
```